pom.xml

```diff
@@ -1871,6 +1872,25 @@
         <artifactId>paranamer</artifactId>
         <version>${paranamer.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.arrow</groupId>
```

Comment:
This is just the dependency management section in the main pom.  The only actual dependency is in spark-sql.